pkg/instance/instance_integration_test.go

```diff
@@ -61,68 +71,143 @@ func TestInstanceHandler(t *testing.T) {
 	groupService := groupService{group: group}
 	stacks := stack.Stacks{
 		"whoami-go": stack.WhoamiGo,
+		"dhis2":     stack.DHIS2,
 	}
 	stackService := stack.NewService(stacks)
 	// classification 'test' does not actually exist, this is used to decrypt the stack parameters
 	helmfileService := instance.NewHelmfileService("../../stacks", stackService, "test")
 	instanceService := instance.NewService(instanceRepo, groupService, stackService, helmfileService)
 
+	s3Dir := t.TempDir()
+	s3Bucket := "database-bucket"
+	err = os.Mkdir(s3Dir+"/"+s3Bucket, 0o755)
+	require.NoError(t, err, "failed to create S3 output bucket")
+	s3 := inttest.SetupS3(t, s3Dir)
+	uploader := manager.NewUploader(s3.Client)
+	s3Client := storage.NewS3Client(s3.Client, uploader)
+	databaseRepository := database.NewRepository(db)
+	databaseService := database.NewService(s3Bucket, s3Client, groupService, databaseRepository)
+
+	authenticator := func(ctx *gin.Context) {
+		ctx.Set("user", &model.User{
+			ID:    user.ID, // TODO: Why is this necessary all of a sudden?
```

Comment:
What fails if you do not set it?

Reply:
Error #01: ERROR: insert or update on table "instances" violates foreign key constraint "fk_instances_user" (SQLSTATE 23503)

That's what I'm getting locally for both tests

Reply:
But to expedite things here, let's just add it